.circleci/config.yml

```diff
@@ -515,7 +530,7 @@ jobs:
           name: Build
           command: |
             cd samples/go/referenceapp
-            go get github.com/godaddy/asherah/go/appencryption@${CIRCLE_BRANCH}
+            go mod edit -replace github.com/godaddy/asherah/go/appencryption=../../../go/appencryption
```

Comment:
Should update the referenceapp's `go.mod` to reference to latest version instead of using replace. This would be consistent with the java pom updates as well.

Reply:
referenceapp should ideally use the current AE version, because no AE updates should break the referenceapp. 

The jar for java and the nuget package for C# are locally installed, so we can make that happen but wasn't able to find a similar method for Go.

Reply:
> referenceapp (and possibly cltf) should ideally use the current AE version, because no AE updates should break the referenceapp.

I think this holds true for the cross-language tests but not so much for the reference apps. The cltf aims to ensure changes in one implementation don't impact cross-language compatibility, so always testing against the latest AE (current branch) makes sense because we want discover issues asap, prior to releasing a new version.

The reference apps, however, serve a different purpose, they exist as a guide for developers looking to use the SDK. And because users of the SDK are encouraged to use the latest released version (as opposed to `master`), our CI pipeline should also follow suit.